elasticsearch/src/test/java/com/tcdi/zombodb/query_parser/TestQueryRewriter.java

```diff
@@ -148,6 +148,17 @@ public Map answer(InvocationOnMock invocationOnMock) throws Throwable {
                             fieldProps.put("analyzer", "fulltext");
 
                             fieldProps = new HashMap();
+                            props.put("sent", fieldProps);
+                            fieldProps.put("analyzer", "date");
+
+                            fieldProps = new HashMap();
+                            props.put("sent", fieldProps);
+                            fieldProps.put("analyzer", "date");
+                            final Map fieldsMap = new HashMap();
+                            fieldsMap.put("date", "something about date");
+                            fieldProps.put("fields", fieldsMap);
```

Comment:
Cool, I was previously using spring-data-es and it wasn't so hard to fire up a little ES node for tests but it sounds like you have it figured out.